pkg/controller/replication/replication_controller.go

```diff
@@ -678,14 +664,14 @@ func (rm *ReplicationManager) syncReplicationController(key string) error {
 
 	var manageReplicasErr error
 	if rcNeedsSync && rc.DeletionTimestamp == nil {
-		manageReplicasErr = rm.manageReplicas(filteredPods, &rc)
+		manageReplicasErr = rm.manageReplicas(filteredPods, rc)
 	}
 	trace.Step("manageReplicas done")
 
 	newStatus := calculateStatus(rc, filteredPods, manageReplicasErr)
```

Comment:
@deads2k I don't really want to merge until after we're close to agreement on how the per-listener resync periods will work for shared informers. If we can't get that, this will have to wait. I'm still wrapping that up.